samples/config/prod/m-cluster/ctsstore-local.yaml

```diff
@@ -1,27 +1,38 @@
 image:
   #repository: forgerock-docker-public.bintray.io/ds
   repository: gcr.io/engineering-devops/ds
-  tag: 6.5.0-M129.1-TTL-SL
+  tag: 6.5.0-M129.2-CTS-TTL
   pullPolicy: Always
 
 instance: ctsstore-local
 
+userstore:
```

Comment:
And this?

Reply:
This has always been there in the original charts values.yaml.  I am only exposing it now in m-cluster.